platform/viewer/src/connectedComponents/ToolContextMenu.js

```diff
@@ -1,9 +1,6 @@
-import React, { Component } from 'react';
+import React, { useRef } from 'react';
 import PropTypes from 'prop-types';
-import cornerstone from 'cornerstone-core';
-import cornerstoneTools from 'cornerstone-tools';
-// This whole component should live in the Measurements Extension :thinking:
-import getMeasurementLocationCallback from '../appExtensions/MeasurementsPanel/getMeasurementLocationCallback';
+import { commandsManager } from './../App.js';
```

Comment:
Not as big of a deal, because this is in the viewer project, but might make more sense to identify the actions that trigger commands, expose them as props, and then bind props to appropriate command callbacks.

Reply:
Makes sense, the same with providers usage of commandsManager.

Reply:
Is `ToolContextMenu` a flavor of `ContextMenu` 🤔

Reply:
ContextMenu is the provider and ToolContextMenu the current component, its very related to tool but I changed the provider name to be just contextmenu, should i update to follow the component name?